packages/realm/src/app-services/SyncConfiguration.ts

```diff
@@ -159,6 +169,7 @@ export function toBindingSyncConfig(config: SyncConfiguration): binding.SyncConf
     customHttpHeaders,
     flxSyncRequested: !!flexible,
     ...parseClientResetConfig(clientReset, onError),
+    cancelWaitsOnNonfatalError: cancelWaitOnNonFatalError,
```

Comment:
Is this difference in name intentional? Could it be the same to avoid confusion?

```suggestion
    cancelWaitsOnNonfatalError: cancelWaitOnNonFatalError,
```

Reply:
It's intentional, I thought that it sounded a little better without the "s", but I don't have a strong opinion about, I'll change it